.yarnrc.yml

```diff
@@ -0,0 +1,15 @@
+# compression does not really make sense with node modules
+compressionLevel: 0
+
+# react native does not yet support pnp https://yarnpkg.com/features/pnp#incompatible
```

Comment:
Does it make sense to create an issue to change this once react-native does support it?

Reply:
I think this will will still take some time up until rn will support this, therefore, I don't see us creating an issue for this. But it would definitely make sense to use pnp, when it is possiple.